### PR TITLE
Update Trivy version

### PR DESCRIPTION
**Pull request by jaroslawgajewski**
Update action to use up to date Trivy version

### CODE REVIEW

File: test/data/yamlconfig.test [hunk 1]
```diff
@@ -77,6 +77,7 @@
           },
           "Description": "libfetch before 2021-07-26, as used in apk-tools, xbps, and other products, mishandles numeric strings for the FTP and HTTP protocols. The FTP passive mode implementation allows an out-of-bounds read because strtol is used to parse the relevant numbers into address bytes. It does not check if the line ends prematurely. If it does, the for-loop condition checks for the '\\0' terminator one byte too late.",
           "Severity": "CRITICAL",
+          "Status": "fixed",
```
Comment (jaroslawgajewski) [inline test/data/yamlconfig.test:80]: @simar7 not sure why but my WFs are complaining for that line. Any advise why?

File: .github/workflows/bump-trivy.yaml [hunk 2]
```diff
@@ -1,12 +1,12 @@
-name: Bump trivy
+name: Bump rivy
```
Comment (simar7) [inline .github/workflows/bump-trivy.yaml:1]: typo?

File: test/data/yamlconfig.test (hunk 1 above)
Comment (simar7) [reply test/data/yamlconfig.test:80]: what's WFs?

**Comment by simar7** [comment]
Looks like you'll have to update some testfiles as well.

**Comment by simar7** [comment]
Closing in favor of https://github.com/aquasecurity/trivy-action/pull/256

**Pull request closed without merging**